### DISCUSSION TITLE
v4.2.2

### DISCUSSION THREAD

**Discussion by michaelliunsky** [michaelliunsky/niRvana-theme#11] [General]
- 主题兼容了 WEBP 格式
- 添加主题图片灯箱

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelliunsky/niRvana-theme/releases/tag/v4.2.2'>v4.2.2</a>.</em>

**Comment by a97115038** [comment]
Without language packs, the theme seems perfect,

**Reply by michaelliunsky** [reply]
Sorry,I can't understand you.
Do you mean that I should add English or other languages' support?

**Reply by a97115038** [reply]
1.//Theme has errors in 6.2 and php8.0.
2.//In fact, there is no need to support multiple languages, at least you need to extract niRvana-theme.po and niRvana-theme.mo If the theme has po and mo, other friends like your theme, I think they are willing to add their own language

**Reply by a97115038** [reply]
Running on an older version WordPress4.7🤦

**Reply by michaelliunsky** [reply]
Maybe I will fix that in theme V5.0
You can try **Wordpress 5.3**
I have tested the theme in that version.
